### PR TITLE
[13.0][MIG] sale_stock_return_request: Migration to v13.0

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc@Tecnativa TT27473

Depends on:
- [x] https://github.com/OCA/stock-logistics-workflow/pull/788

### CODE REVIEW

**Comment by CarlosRoca13** [comment]
https://github.com/OCA/stock-logistics-workflow/pull/788 already merged, can you do a rebase?

**Comment by sergio-teruel** [comment]
@CarlosRoca13 Changes done!!

File: sale_stock_return_request/README.rst [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to *Purchases > Purchase > Return Request* and place a new return.
```
Comment (CarlosRoca13) [inline sale_stock_return_request/README.rst]:
```suggestion
#. Go to *Inventory > Operations > Return Request* and place a new return.
```

**Review by CarlosRoca13 — COMMENTED** [review]
Functional and code review! Is it still WIP?

**Comment by sergio-teruel** [comment]
@CarlosRoca13 changes done!

**Review by CarlosRoca13 — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by pedrobaeza — APPROVED** [review]
Code review

**Pull request merged**